### PR TITLE
Tests added

### PR DESCRIPTION
**Pull request by AlinaZhykharieva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "api-axios-mocha",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ovartem) [inline package.json]: seems like npm t won't run tests :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "author": "Alina Zhykharieva",
+  "license": "ISC",
+  "dependencies": {},
+  "devDependencies": {
+    "axios": "^0.21.1",
+    "chai": "^4.3.3",
+    "jsonschema": "^1.4.0",
```
Comment (ovartem) [inline package.json]: why do you need this package?

File: test/users.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    });
+
+    it(`response header should has "${CONTENT_TYPE}" ${HEADER_PROPERTY}`, async () => {
+        const header = (await axios.get(env.uri)).headers
+        expect(header.hasOwnProperty(HEADER_PROPERTY)).to.be.true;
+        expect(header[HEADER_PROPERTY]).eql(CONTENT_TYPE);
+    });
+
+    it(`response body should have array with length ${RESPONSE_BODY_LENGTH}`, async () => {
+        expect(((await axios.get(env.uri)).data).length).eql(RESPONSE_BODY_LENGTH);
```
Comment (ovartem) [inline test/users.js]: nitpicking `data).length`  - you don't really need this `)`

Comment (AlinaZhykharieva1) [reply test/users.js]: That was in the task: the content of the response body is the array of 10 users. Maybe I got it wrong?

**Pull request merged**